### DISCUSSION TITLE
replace net/http to fasthttp

### DISCUSSION THREAD

**Discussion by passwa11** [projectdiscovery/httpx#744] [Ideas]
### Please describe your feature request:

helo ,man do u have any plan to replace net/http to [fasthttp](https://github.com/valyala/fasthttp) ?
it's more fast than the net/http !

**Comment by ehsandeep** [comment]
@passwa11 thanks; I'm moving this to the discussion section to keep this open.

**Reply by Mr-xn** [reply]
Has anyone tested them?
